### DISCUSSION TITLE
链接失败或超时有啥解决办法吗

### DISCUSSION THREAD

**Discussion by LTHPKBTE** [Samarium150/mirai-console-lolicon#25] [Q&A] [answered]
广东移动网，每次都是链接超时，但是看起来请求是已经发过去了的，多请求亿次才会有一个获取到图片....有啥好的解决办法吗

**Comment by Samarium150** [comment]
是图片的请求失败还是api调用的失败?

**Reply by LTHPKBTE** [reply]
看起来API调用没问题，后面就报链接超时了

**Reply by Samarium150** [reply]
看起来国内要从i.pixiv.cat拿图比较困难, 最简单的方法是把bot扔在境外服务器, 我出个补丁加个自定义超时时间

**Reply by Samarium150** [reply]
#27 中增加了默认的超时时间和图片压缩选项, 可以看新的[配置文件](https://github.com/Samarium150/mirai-console-lolicon/wiki/Usage#configuration%E9%85%8D%E7%BD%AE)说明

**Reply by LTHPKBTE** [reply]
![图片](https://user-images.githubusercontent.com/53574505/105955533-6a126380-60b1-11eb-868e-6855f1673c00.png)
貌似这个链接不会走代理？我想让他走代理有啥办法吗

**Comment by LTHPKBTE** [comment, marked as answer]
已解决。hosts文件加一行解析即可
i.pixiv.cat 172.67.141.190

**Reply by LTHPKBTE** [reply]
用这个IP能连上，速度较慢(一张1.38 MB的图片消耗10.24 秒载入）但是能用了
